lib/tnt_api/tracking_by_consignment_response.rb

```diff
@@ -0,0 +1,8 @@
+module TNTApi
+  class TrackingByConsignmentResponse
+
+    def initialize(response)
+      response.body[:tracking_by_consignment_response][:parcel]
```

Comment:
What's going on here? You are doing nothing with the response, it will be lost.  it should be assigned to a variable so we can read it, and this behaviour should be tested.